recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - flask-login ==0.2.11
     - flask-swagger ==0.2.13
     - flask-wtf ==0.12
-    - funcsigs ==1.0.0  # [py<33]
+    - funcsigs ==1.0.0   [py<33]
```

Comment:
The `#` is not really a comment in this case, but a [selector]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors ). So it was working correctly before.

Reply:
You're right. I'm a bit puzzled though, I tried to conda install airflow on a fresh python 3.5 conda environment and it was complaining about not finding funcsigs==1.0.0 which looks bizarre to me. 
However, I cannot reproduce the error with a machine with an internet connection... so I guess you can drop the PR. Thx for checking in any case.